<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>539</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文细黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="uartGroupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxPortName</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxPortName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxBaudRate">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>7</number>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxBaudRate</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxStopBit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxStopBit</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDataBit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxDataBit</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxParity">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>无校验</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>偶校验</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>奇校验</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>奇偶校验</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxParity</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenPort">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGetPortName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textRecvDataHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="recvGroupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>260</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>接收设置</string>
    </property>
    <widget class="QCheckBox" name="chkBoxShowTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>64</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>时间</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkBoxShowSendData">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>64</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearRecvData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>清除显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>保存文件</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkBoxShowIndex">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>64</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>序号</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtSendData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="recvGroupBox_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>350</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <widget class="QCheckBox" name="chkBoxSendRepeat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>重复发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearSendData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>清除发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-发送</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRepeatCnt">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="次数"/>
     </property>
     <property name="suffix">
      <string>次</string>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbtnHexMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>十六进制</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnTextMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>文本</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>87</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重复间隔mS</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxPortName</cstring>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxRepeatInterval">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="次数"/>
     </property>
     <property name="suffix">
      <string>mS</string>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCommandLinkButton" name="btnRxTxClear">
     <property name="geometry">
      <rect>
       <x>721</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
     <property name="icon">
      <iconset theme="none">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>161</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="labelRxCnt">
        <property name="font">
         <font>
          <family>华文细黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RX: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelTxCnt">
        <property name="text">
         <string>TX: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textRecvData">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>291</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="About"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="Save">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="Connect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="DisConnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
